app/views/layouts/application.html.erb

```diff
@@ -1,7 +1,9 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>LittleEstyShop7</title>
+    <header>
+      <h1> Little Esty Shop </h1>
+    </header>
```

Comment:
In general, the `<head>` tags in an HTML file are for metadata and loading Javascript/CSS files. 

If we want to add "Lil Esty Shop" to every page, that should live inside the body.